motion/dp_controller/src/quaternion_dp_controller.cpp

```diff
@@ -0,0 +1,136 @@
+/*   Written by Kevin Strandenes and Anders Slåkvik, Student
+     Documentation written by Kevin Strandenes and Anders Slåkvik
+     Copyright (c) 2023 Beluga AUV, Vortex NTNU.
+     All rights reserved. */
+
+#include "dp_controller/quaternion_dp_controller.h"
+#include <math.h>
+
+QuaternionPIDController::QuaternionPIDController() { // float W, float B,
+                                                     // Eigen::Vector3d r_G,
+                                                     // Eigen::Vector3d r_B){
+  m_B = 23 * 9.81;
+  m_W = 24 * 9.81;
```

Comment:
Magic numbers!! As an example, even though most people will likely recognize 9.81, it is still much better to define `float gravity = 9.81` here. 

Either make this more explicit or just set them to zero, since these should be set from the outside anyways